src/main/java/cloud/eppo/BaseEppoClient.java

```diff
@@ -91,7 +112,7 @@ private EppoHttpClient buildHttpClient(
   }
 
   protected void loadConfiguration() {
-    requestor.load();
+    requester.load(expectObfuscatedConfig);
```

Comment:
nit: it seems like we could pass `expectObfuscatedConfig` to requester constructor. The expectation never changes in runtime and it's not used anywhere else beside the requester.

Reply:
I had thought to do this but missed it in the last pass. thanks!